### PR TITLE
ATB-215: Removed duplication of ../../config

### PR DESCRIPTION
**Pull request by jessica-li-01**
## Proposed changes
Removed the duplication of ../../config as it was not needed

### What changed

Deleted the additional ../config and added the same reference in the original config line.

### Why did it change

Unnecessary duplication

### Issue tracking

- [ATB-215](https://govukverify.atlassian.net/jira/software/c/projects/ATB/boards/241?modal=detail&selectedIssue=ATB-215&assignee=635679e1fc0cc7a600ae36d5)

### Environment variables or secrets
- [x] No environment variables or secrets were added or changed

- [ ] [Application configuration](https://govukverify.atlassian.net/TODO) is up-to-date
- [ ] Documented in the README
- [ ] Added to deployment steps
- [ ] Added to local startup config

- [ ] Let the tech lead know if the Dockerfile has changed

### Other considerations
- [x] Demo to a BA, TA, and the team.
- [ ] Recorded demo shared on [#govuk-accounts-tech](https://gds.slack.com/archives/C011Y5SAY3U)
- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[ATB-215]: https://govukverify.atlassian.net/browse/ATB-215?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ATB-215]: https://govukverify.atlassian.net/browse/ATB-215?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/assets/javascript/application.js [hunk 1]
```diff
@@ -16,7 +16,9 @@ function initChangePhoneNumber() {
     intPhoneNumberCheckbox.addEventListener("click", function(event) {
       onIntNumberSelected(event.currentTarget, phoneNumberInput);
     });
-    window.addEventListener("load", onIntNumberSelected(intPhoneNumberCheckbox, phoneNumberInput));
+    window.addEventListener("load", function(event) {
```
Comment (TaherKapasi) [inline src/assets/javascript/application.js:19]: This change should be in separate PR for (ATB-206)[https://govukverify.atlassian.net/browse/ATB-206]

**Review by TaherKapasi — CHANGES_REQUESTED** [review]

File: src/assets/javascript/application.js (hunk 1 above)
Comment (jessica-li-01) [reply src/assets/javascript/application.js]: Separated :)

**Review by KG700 — APPROVED** [review]

**Review by DougsCER — APPROVED** [review]
